cli/tests/test_cmd_upgrade.py

```
@@ -6705,6 +6705,67 @@ class TestUpgradeManifest(unittest.TestCase):
     """Release-owned upgrade manifests let future versions declare upgrade
     policy even when the local upgrade script is older than the release."""
 
+    @staticmethod
+    def _native_windows_acl_fixture():
```

Ruff (0.16.0)

[warning] Missing return type annotation for staticmethod `_native_windows_acl_fixture`

(ANN205)